client/src/main/java/com/defold/extender/client/ExtenderClient.java

```diff
@@ -267,12 +266,19 @@ private void build_async(String platform, String sdkVersion, MultipartEntity ent
                     response.getEntity().writeTo(new FileOutputStream(destination));
                 } else {
                     log("Job %s did not complete successfully. Writing log to %s", jobId, log);
-                    response.getEntity().writeTo(new FileOutputStream(log));
-                    throw new ExtenderClientException("Failed to build source.");
+                    OutputStream os = new FileOutputStream(log);
+                    os.write(String.format("Job id: %s; traceId: %s", jobId, traceId == null ? "null" : traceId).getBytes());
```

Comment:
Write to the log file information about `jobId` and `traceId` to easier debug user's requests about failed builds.